core/work_ledger.py

```
@@ -47,9 +47,12 @@ def to_dict(self) -> Dict:
 class WorkLedger:
     """按会话维护工作账本。"""
 
-    def __init__(self, retain_finished: int = 8):
+    def __init__(self, retain_finished: int = 8, running_timeout: float = 300.0):
         self._items: Dict[str, Dict[str, WorkItem]] = {}
         self.retain_finished = max(1, int(retain_finished))
+        # running 超时自动失效：防止 complete_work 漏触发（如流式回复/主脑失败）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)